docusaurus/docs/packages/pkg/client/pkg.md

```diff
@@ -113,21 +112,278 @@ txctx -."tx query".-> chain
 go get github.com/pokt-network/poktroll/pkg/client
 ```
 
-## Usage
+## Usage (Supplier Client)
+
+### Basic Example (CLI command)
+
+```go
```

Comment:
Starting a discussion. 

I understand why we're providing such long examples in the docs, but I generally do not think it's a good idea (unless you're a bigger org with lots of users that'll maintain it). 

IMO:
1. This will get very quickly outdated (potentially not compile) and raise questions for readers
2. Linking to an example file / code example in the code:
	- Achieves the same thing from an onboarding perspectives
	- Ensures refactoring changes continue compiling
	- Makes .md documents "less scary"
3. From experience, I've found that small (< 10 lines) code examples in `.md` files have a longer lifespan than full on examples like this.

Reply:
Ditto - example_test.go files are the way to go for this like we did with the replay client, as they are actual code they will break on changes - making us keep them up to date.

Reply:
Agreed; however, unfortunately, we cannot make these into [testable examples](https://go.dev/blog/examples) without making them even larger and unwieldy because they would be integration examples (i.e. dependent on in-memory networks).

Reply:
@bryanchriswhite Lmk wyt